dataHandling/handling.py

```diff
@@ -1,8 +1,8 @@
-def print_handling():
-    print("Hello from handling")
 class dataHandler:
     def __init__(self, argument1, argument2):
         return
+    def __init__(self):
+        return
 
     def print_handling():
```

Comment:
Add "self" to arguments to fix error:
"function takes 0 positional arguments but 1 was given"